ansible_base/jwt_consumer/common/auth.py

```diff
@@ -105,13 +105,10 @@ def parse_jwt_token(self, request):
         is_cached, user_defaults = self.cache.check_user_in_cache(self.token)
```

Comment:
```suggestion
        _, user_defaults = self.cache.check_user_in_cache(self.token)
```
If we're dropping the `is_cached` check